Ensure tmp directory exists

jia1 opened the pull request:
1. At the start function: mkdir if tmp does not exist
2. Alternative: Use .keep

Signed-off-by: Jia Yee <jiayeerawr@gmail.com>

jia1 committed:
Ensure tmp directory exists

1. At the start function: mkdir if tmp does not exist
2. Alternative: Use .keep

Signed-off-by: Jia Yee <jiayeerawr@gmail.com>

thenaesh reviewed (changes requested):
@jia1 I think we can remove the extra echo statements, for consistency with the rest of the statements. Creating a nonexistent directory is a routine matter and is not really an error condition.

Also, perhaps rename `checkTmp` to `createTmpDirectoryIfNotExists`?

jia1 committed:
Rename check method and remove echo

1. Rename checkTmp to a more useful name -
createTmpDirectoryIfNotExists
2. Remove additional echo statements in (1)

Signed-off-by: Jia Yee <jiayeerawr@gmail.com>
